protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -23,6 +23,14 @@ type binding struct {
 	*descriptor.Binding
 }
 
+// GetBinding returns the binding body
+func (b binding) GetBinding() string {
+	if b.Body.FieldPath.String() != "" {
+		return b.Body.FieldPath.String()
+	}
+	return "*"
```

Comment:
is it safe to assume that an empty `FieldPath` means `*`? It seems reasonable to me, but I'm wondering if returning the empty string in this case makes more sense.